pkg/driver_cryptocell/psa_cc/hashes_sha1.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2021 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_psa_crypto pkg_driver_cryptocell
+ * @{
+ *
+ * @brief       Glue code translating between PSA Crypto and the CryptoCell 310 driver APIs
+ *
+ * @author      Lena Boeckmann <lena.boeckmann@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include "kernel_defines.h"
+#include "psa_periph_hashes_ctx.h"
+#include "psa_periph_hashes_common.h"
+
+#define ENABLE_DEBUG    (0)
```

Comment:
```suggestion
#define ENABLE_DEBUG    0
```

There was a PR a year ago that went through all the code base to makw these defines co consistent. Since `IS_ACTIVE ()` doesn't work with braces, it was decides to omit them here consistently.

Reply:
Done